Index.html

```diff
@@ -217,27 +220,29 @@ <h1 class="bottom-line">Skills</h1>
         <hr>
     </ul>
     </article>
-    <article class="header-img">
+
+    <footer class="header-img">
         <h1 class="index-h1">Contact me</h1>
-        <p>If you have an application you are interested in developing,
+        <p class="text">
+            If you have an application you are interested in developing,
            a feature that you need built or a project that needs coding.
            I'd love to help with it 
         </p>
-        <form class="form-control">
+        <form action="https://formspree.io/f/{form_id}" method="POST">
             <div class="pl-0 form-group">
                 <label for="name"></label>
-                <input class="field-spacing" type="text" name="name" placeholder="name">
+                <input class="field-spacing" type="text" name="name" placeholder="name" required>
```

Comment:
- Great Job adding the "required" attribute to all your input fields 👍🏽 

- I think your input tag for name currently lacks a maximum length.  This will help limit the number of alphabets per name inputed by a user. Can we kindly add `maxlength=30` to your name field? 

- Also, it is important to include a maximum length to your textarea as well. Kindly include a `maxlength=500` to your textarea. 🍒